tools/somalier_relate.cwl

```diff
@@ -0,0 +1,56 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: somalier-relate
+doc: "Tool to calculate sample relatedness and grouping"
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 1000
+    coresMin: 4
+  - class: DockerRequirement
+    dockerPull: 'brentp/somalier:v0.2.19'
+baseCommand: [somalier, relate]
+
+inputs:
+  somalier_output: { type: 'File[]', doc: "Somalier-formatted variant files",
+    inputBinding: { position: 2} }
+  groups: { type: 'File?', doc: "optional path  to expected groups of samples (e.g. tumor normal pairs).
```

Comment:
Think people are gonna freak out over this file? It's kinda odd.

Reply:
The groups file? Why would they freak out?